app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elf: list[Elf]) -> None:
```

Comment:
elf is not the most suitable name - as a convention in python we use names with plural forms for lists or other collections of data. Or you can add suffix `_list` in this particular case.